### ISSUE TITLE
RetroArchHosts not recognized

### ISSUE THREAD

**Issue by MarcNuernberger** [Skarsnik/QUsb2snes#118]
RetroArchHosts variable in config does not get recognized on application launch.

OS: Fedora Workstation 36
QUsb2Snes Version: 0.7.23.2
QT Version: 5.15.3

QUsb2Snes.conf:
```
[General]
FirstTime=true
RetroArchHosts="remoteName=192.168.123.123"
debugLog=true
retroarchdevice=true
```

log-debug.txt:
```
2022-07-17T23:03:01              default - Debug: Runing QUsb2Snes version  "0.7.23.2"
2022-07-17T23:03:01              default - Debug: Compiled against Qt 5.15.3 , running 5.15.3
2022-07-17T23:03:01             WSServer - Debug: WebSocket server started : listenning  QHostAddress(QHostAddress::Any) port :  8080
2022-07-17T23:03:01             WSServer - Debug: WebSocket server started : listenning  QHostAddress(QHostAddress::Any) port :  23074
```

After these log entries nothing else is happening. In the GUI the "Devices" entry is empty and not clickable.

**Comment by Skarsnik** [comment]
Weird, what the diagnostic window shows?

**Comment by MarcNuernberger** [comment]
Sorry for the delay. Here is a screenshot of the Diagnostic Tool window.

![Screenshot from 2022-07-18 12-46-00](https://user-images.githubusercontent.com/3965719/179496058-2deea7c0-22e8-48a7-9e63-faca8132d361.png)

I assume a permission issue here. If I start QUsb2Snes with sudo, I can see the device list but still don't get a connection. Even with a locally running RetroArch it won't work.

**Comment by Skarsnik** [comment]
This is weird. Did you activated the network command support in ra?

**Comment by MarcNuernberger** [comment]
Yes I did that on both, the local and the remote RetroArch instances.

**Comment by Skarsnik** [comment]
Is there new entry on the debug log when you open the diagnostic window?
I will have a look at qusb on linux maybe this Sunday at best ~~
